x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elastic/ElasticInferenceService.java

```diff
@@ -104,15 +116,41 @@ public ElasticInferenceService(
         this.modelRegistry = Objects.requireNonNull(modelRegistry);
         this.authorizationHandler = Objects.requireNonNull(authorizationHandler);
 
-        configuration = new Configuration(enabledTaskTypesRef.get());
+        configuration = new Configuration(authRef.get().authorizedTaskTypesAndModels.getEnabledTaskTypes());
+        defaultModels = initDefaultEndpoints(elasticInferenceServiceComponents);
 
         getAuthorization();
     }
 
+    private static Map<String, Model> initDefaultEndpoints(ElasticInferenceServiceComponents elasticInferenceServiceComponents) {
+        return Map.of(
+            DEFAULT_CHAT_COMPLETION_MODEL_ID_V1,
+            new ElasticInferenceServiceCompletionModel(
+                DEFAULT_CHAT_COMPLETION_ENDPOINT_ID_V1,
+                TaskType.CHAT_COMPLETION,
+                NAME,
+                new ElasticInferenceServiceCompletionServiceSettings(DEFAULT_CHAT_COMPLETION_MODEL_ID_V1, null),
+                EmptyTaskSettings.INSTANCE,
+                EmptySecretSettings.INSTANCE,
+                elasticInferenceServiceComponents
+            )
+        );
+    }
+
+    private record AuthorizedContent(
```

Comment:
Just an aggregation of all the different pieces we need to expose (enabled task types, DefaultConfigId objects, and a list of models).